### ISSUE TITLE
Can this be used for OpenAI models?

### ISSUE THREAD

**Issue by jlia0** [noamgat/lm-format-enforcer#26]
Since gpt-3.5-turbo-instruct and new releases on gpt-4-turbo, is it possible to use lm-format-enforcer with these models?

**Comment by noamgat** [comment]
As stated in the readme:


# Known issues and limitations
- LM Format Enforcer requires a python API to process the output logits of the language model. This means that until the APIs are extended, it can not be used with OpenAI ChatGPT and similar API based solutions.
